environments.json

```diff
@@ -1381,6 +1381,13 @@
     "release": "2017-03-27",
     "obsolete": false
   },
+  "safari11": {
+    "full": "Safari 11",
+    "family": "JavaScriptCore",
+    "short": "SF 11",
+    "release": "2017-06-05",
+    "unstable": false
```

Comment:
Was it released already?  I was under the impression that it is still in beta...